Sources/AlamofireSessionRenewer/AlamofireSessionRenewer.swift

```diff
@@ -8,105 +8,170 @@
 import Foundation
 import Alamofire
 
-public typealias SuccessRenewHandler = (String) -> Void
-public typealias FailureRenewHandler = (Bool) -> Void
-public typealias RenewCredentialHandler = ((@escaping SuccessRenewHandler, @escaping FailureRenewHandler) -> Void)
+public typealias SuccessRenewHandler = (String) async -> Void
+public typealias FailureRenewHandler = (Bool) async -> Void
+public typealias RenewCredentialHandler = ((SuccessRenewHandler, FailureRenewHandler) async -> Void)
 
 /// This class is responsible for authentication credentials renewing process
-open class AlamofireSessionRenewer: RequestInterceptor {
+final public actor AlamofireSessionRenewer: RequestInterceptor {
+    
+    // MARK: Properties
     
     /// Error code indicating request is missing authentication info. Defaults to HTTP code 401
-    public let authenticationErrorCode: Int
+    private let authenticationErrorCode: Int
     
     /// Authentication information request header field name
-    public let credentialHeaderField: String
+    private let credentialHeaderField: String
     
     /// Number of times to retry credentials renewing process upper limit
-    public let maxRetryCount: UInt?
+    private let maxRetryCount: UInt?
+    
+    /// Error domain indicating request we should retry
+    private let errorDomain: String
     
     /// Authentication information unit
-    open var credential: String?
+    private var credential: String?
     
     /// Queue which stores requests to retry
```

Comment:
nit: that comment isn't accurate anymore. Let's replace it with:
```suggestion
    /// An array which stores closures to retry requests after updating credentials
```